authors/apps/articles/serializers.py

```diff
@@ -5,7 +5,6 @@ class ArticleSerializer(serializers.ModelSerializer):
     """This serializer requests and creates a new article"""
     class Meta:
         fields = [
-            'id',
             'slug',
```

Comment:
@AtamaZack NiceWork i see you have the read_only_field empty you could add the slug and author . or just remove it if there is no field that needs to be added there.

Reply:
Thanks, on it.